python/flask/security/xss/audit/direct-use-of-jinja2.yaml

```diff
@@ -15,7 +15,9 @@ rules:
   - python
   severity: WARNING
   pattern-either:
+  - pattern: import jinja2
```

Comment:
Do we want to fire on import?

Reply:
I dunno, I got scared that I'd miss some other case 😐; I think for Flask, the recommendation would be to always prefer `render_template` to using `jinja2` in any capacity. What do you think?